src/test/java/core/basesyntax/ValidationTest.java

```diff
@@ -0,0 +1,264 @@
+
+    private static final String VALID_LOGIN_FIRST = "firstDefaultLogin";
+    private static final String VALID_LOGIN_SECOND = "secondDefaultLogin";
+    private static final String VALID_LOGIN_THIRD = "thirdDefaultLogin";
+
+    private static final String VALID_PASSWORD_FIRST = "123456";
+    private static final String VALID_PASSWORD_SECOND = "LOOOOOOOOOOOOONG";
+    private static final String VALID_PASSWORD_THIRD = "qwe123ZXC$%^";
+
+    private static final int VALID_AGE_FIRST = 18;
+    private static final int VALID_AGE_SECOND = 19;
+    private static final int VALID_AGE_THIRD = 91;
+
+    private static RegistrationServiceImpl registrationService;
+    private static List<User> validUserData;
+    private static StorageDaoImpl storageDaoImpl;
+
+    private User validUserBob;
+    private User validUserAlice;
+    private User validUserJohn;
+    private User validUserSecondBob;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDaoImpl = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUserBob = new User();
+        validUserBob.setLogin(VALID_LOGIN_FIRST);
+        validUserBob.setPassword(VALID_PASSWORD_FIRST);
+        validUserBob.setAge(VALID_AGE_FIRST);
+
+        validUserSecondBob = new User();
+        validUserSecondBob.setLogin(validUserBob.getLogin());
+        validUserSecondBob.setPassword(validUserBob.getPassword());
+        validUserSecondBob.setAge(validUserBob.getAge());
+
+        validUserAlice = new User();
+        validUserAlice.setLogin(VALID_LOGIN_SECOND);
+        validUserAlice.setPassword(VALID_PASSWORD_SECOND);
+        validUserAlice.setAge(VALID_AGE_SECOND);
+
+        validUserJohn = new User();
+        validUserJohn.setLogin(VALID_LOGIN_THIRD);
+        validUserJohn.setPassword(VALID_PASSWORD_THIRD);
+        validUserJohn.setAge(VALID_AGE_THIRD);
+
+        validUserData = new ArrayList<>();
+        validUserData.add(validUserBob);
+        validUserData.add(validUserAlice);
+        validUserData.add(validUserJohn);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_checkPrivateFields_Ok() {
+        List<Field> privateFields = new ArrayList<>();
+        List<Field> allFields = Arrays.asList(RegistrationServiceImpl.class.getDeclaredFields());
+        for (Field field : allFields) {
+            if (Modifier.isPrivate(field.getModifiers())) {
+                privateFields.add(field);
+            }
+        }
+        Assertions.assertEquals(allFields.size(), privateFields.size(),
+                "You should have private fields only!" + '\n');
+        Assertions.assertTrue(privateFields.size() >= allFields.size(),
+                "You should have at least " + allFields.size() + " fields" + '\n');
+    }
+
+    @Test
+    void register_ValidUsers_Ok() throws RegistrationException {
```

Comment:
do not declare RuntimeException 
```suggestion
    void register_ValidUsers_Ok() {
```

Reply:
Oh yes, an artifact from a previous version.